modules/assignment/src/main/java/com/intuit/wasabi/assignment/AssignmentsModule.java

```diff
@@ -100,6 +111,42 @@ protected void configure() {
         LOGGER.debug("installed module: {}", AssignmentsModule.class.getSimpleName());
     }
 
+    private void bindMetadataCache(final Properties properties) {
+        Boolean metadataCacheEnabled = Boolean.parseBoolean(getProperty("metadata.cache.enabled", properties, "true"));
+        bind(Boolean.class).annotatedWith(named("AssignmentsMetadataCacheEnabled")).toInstance(metadataCacheEnabled);
+
+        if(metadataCacheEnabled) {
+            //This is a cache refresh interval, at this frequency cache will be refreshed.
+            Integer metadataCacheRefreshInterval = Integer.parseInt(getProperty("metadata.cache.refresh.interval", properties, "5"));
```

Comment:
Could be, but this cache is something we would never want to be refreshed at the intervals of SECONDS or HOURS but only by MINUTES so my thoughts just keeping it at MINUTES precisions avoids any confusion or user errors.